### PR TITLE
feat(components): new highlighter for Emphasis

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

We are using a more organic "hand-drawn" brushstroke to emphasize segments of our headings in our brand materials (see screenshot). This change brings in the same SVG asset from getjobber.com to provide consistent styling for our highlights.
image

One other consideration instead of referencing the same asset from getjobber.com's asset pipeline was bringing this SVG into a small sub-component. If that's the better way forward, I'm good with that.

## Changes

Change the CSS for `highlight` emphasis

### Changed

- Before: bright accent rectangular underline
image

- After: brushstroke underline that also works on non-Poppins/display elements
image

## Testing

Paste this snippet into the sandboxes in `Emphasis`

```
<Heading level={1}>
  Get paid <Emphasis variation="highlight">faster</Emphasis>
</Heading>
<Heading level={2}>
  Get paid <Emphasis variation="highlight">faster</Emphasis>
</Heading>
<Heading level={3}>
  Get paid <Emphasis variation="highlight">faster</Emphasis>
</Heading>
<Heading level={4}>
  Get paid <Emphasis variation="highlight">faster</Emphasis>
</Heading>
<Heading level={5}>
  Get paid <Emphasis variation="highlight">faster</Emphasis>
</Heading>
<Heading level={6}>
  Get paid <Emphasis variation="highlight">faster</Emphasis>
</Heading>
<Heading level={1}>
  Get paid <Emphasis variation="highlight">faster and faster faster and faster faster and faster faster and faster</Emphasis> than ever before
</Heading>
<Heading level={2}>
  Get paid <Emphasis variation="highlight">faster and faster faster and faster faster and faster faster and faster</Emphasis> than ever before
</Heading>
<Heading level={3}>
  Get paid <Emphasis variation="highlight">faster and faster faster and faster faster and faster faster and faster</Emphasis> than ever before
</Heading>
<Heading level={4}>
  Get paid <Emphasis variation="highlight">faster and faster faster and faster faster and faster faster and faster</Emphasis> than ever before
</Heading>
<Heading level={5}>
  Get paid <Emphasis variation="highlight">faster and faster faster and faster faster and faster faster and faster</Emphasis> than ever before
</Heading>
<Heading level={6}>
  Get paid <Emphasis variation="highlight">faster and faster faster and faster faster and faster faster and faster</Emphasis> than ever before
</Heading>

<Text size="small">
  <Emphasis variation="bold"><Emphasis variation="highlight">Save $240</Emphasis></Emphasis> when you choose our Annual
  Plan
</Text>
<Text>
  <Emphasis variation="bold"><Emphasis variation="highlight">Save $240</Emphasis></Emphasis> when you choose our Annual
  Plan
</Text>
<Text size="large">
  <Emphasis variation="bold"><Emphasis variation="highlight">Save $240</Emphasis></Emphasis> when you choose our Annual
  Plan
</Text>
<Text size="small">
  <Emphasis variation="bold"><Emphasis variation="highlight">Save $240 and so much more than that, really, because these plans are so valuable to you and your small business that their value is arguably incalculable</Emphasis></Emphasis> when you choose our Annual
  Plan
</Text>
<Text>
  <Emphasis variation="bold"><Emphasis variation="highlight">Save $240 and so much more than that, really, because these plans are so valuable to you and your small business that their value is arguably incalculable</Emphasis></Emphasis> when you choose our Annual
  Plan
</Text>
<Text size="large">
  <Emphasis variation="bold"><Emphasis variation="highlight">Save $240 and so much more than that, really, because these plans are so valuable to you and your small business that their value is arguably incalculable</Emphasis></Emphasis> when you choose our Annual
  Plan
</Text>
```

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Typography/css/Emphasis.css [hunk 1]
```diff
@@ -3,16 +3,14 @@
 }
 
 .highlight {
-  margin-left: calc(-1 * var(--space-smaller));
-  padding: 0 var(--space-smaller);
+  display: inline;
```
Comment (chris-at-jobber) [inline packages/components/src/Typography/css/Emphasis.css:6]: The linter gets upset that I'm using display: inline and margin on the same element

**Comment by hosseintalebi** [comment]
I like it! 💯

**Comment by hosseintalebi** [comment]
I really like it!

File: packages/components/src/Typography/css/Emphasis.css [hunk 2]
```diff
@@ -3,16 +3,13 @@
 }
 
 .highlight {
-  margin-left: calc(-1 * var(--space-smaller));
-  padding: 0 var(--space-smaller);
-  font-family: var(--typography--fontFamily-display);
+  display: inline;
+  padding: var(--space-smaller) 0;
```
Comment (rodrigoeidelvein) [inline packages/components/src/Typography/css/Emphasis.css:7]: Do we actually need this padding? In the first image in the PR description the brushstroke looks closer to the text

**Review by rodrigoeidelvein — COMMENTED** [review]
Nice work! I tested it and it looks pretty good!

Just left a comment about the padding

File: packages/components/src/Typography/css/Emphasis.css (hunk 2 above)
Comment (chris-at-jobber) [reply packages/components/src/Typography/css/Emphasis.css:7]: I checked and we do - it prevents the highlight from breaking into the text at smaller sizes:

**Without**
image

**With**
image

Although I wonder if maybe there's a more systematic way of handling it? I think the difference in gap between the smaller size and larger size comes from the "90%" value... Going to try something

Comment (chris-at-jobber) [reply packages/components/src/Typography/css/Emphasis.css]: Update: I tried something and it worked 😄

**Comment by chris-at-jobber** [comment]
Updating to a percentage-based approach for height of the background allows this to scale without padding hacks

image

image

**Pull request merged**